warehouse/models/intermediate/gtfs/gtfs_quality/_int_gtfs_quality.yml

```diff
@@ -411,7 +411,19 @@ models:
     columns: *stg_gtfs_guideline_columns
 
 # --- GTFS-service-data-level
+# ----- Schedule only
 
+  - name: int_gtfs_quality__demand_responsive_routes_match
+    tests: &stg_gtfs_guideline_tests_gtfs_service_data_schedule
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - date
+            - gtfs_service_data_key
+      - dbt_utils.equal_rowcount:
+          compare_model: ref('int_gtfs_quality__gtfs_service_data_schedule_guideline_index')
```

Comment:
Can we add yaml for `int_gtfs_quality__gtfs_service_data_schedule_guideline_index` too? Otherwise this looks good to me

Reply:
Added for this and all of the new index tables!